src/modules/ui/atoms/macaw-ui/FormInput.tsx

```diff
@@ -22,6 +22,7 @@ export function FormInput<
       error={!!fieldState.error?.message}
       {...props}
       {...field}
+      helperText={fieldState.error?.message || props.helperText}
```

Comment:
Are we showing the error message anywhere? `error` doesn't accept any text value, it just makes the form red.
Maybe we should add an error message to the `helperText`?